### PR TITLE
5.2.2 release notes

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: release-notes/release-notes522.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+> This is the {{site.cloudname}} and {{site.enterpriseversion}} release notes. For information on the latest community version of {{site.productname}}, see: [{{site.productname}} Changelog]({{site.baseurl}}/changelog/).
+
+## General bug fixes
+
+{{site.productname}} 5.2.2 provides fixes for the following bugs:
+
+* Fixed an issue where anchors could not be inserted on empty lines.
+* Fixed text decorations (underline, strikethrough) not consistently inheriting the text color.
+* Fixed `format` menu alignment buttons inconsistently applying to images.
+* Fixed the floating toolbar drawer height collapsing when the editor is rendered in modal dialogs.
```
Comment (metricjs) [inline release-notes/release-notes522.md:26]: Found the non-modal dialog case: http://fiddle.tinymce.com/Pfhaab

Comment (metricjs) [reply release-notes/release-notes522.md:26]: See comments on changelog PR

File: release-notes/release-notes522.md [hunk 2]
```diff
@@ -32,10 +38,16 @@ The following premium plugins updates were released alongside {{site.productname
 
 ### PowerPaste 5.2.2
 
-The {{site.productname}} 5.2.3 release includes an accompanying release of the **PowerPaste** premium plugin.
+The {{site.productname}} 5.2.2 release includes an accompanying release of the **PowerPaste** premium plugin.
```
Comment (EkimChau) [inline release-notes/release-notes522.md:41]: would this release bump powerpaste to 5.2.4 ?

I believe its currently 5.2.3
https://github.com/tinymce/tinymce-enterprise/blob/develop/artifacts.js

Comment (lnewson) [reply release-notes/release-notes522.md:41]: That has already been bumped to the version being released.

Note: I also edited the above as that was linking to a private repo in a public PR.

**Pull request merged**